portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase

```diff
@@ -573,7 +557,7 @@ definition {
 		task ("Start Har recording") {
 			ProxyUtil.startHarRecording("blogViewed");
 
-			Pause(locator1 = "5000");
+			ACUtils.waitForSendingRequest();
```

Comment:
Hey @calvinctvo, I think you could put this macro for after the iteration in DXP. I think you could put. What do you think?

```
task ("Wait for sending requests") {
	ACUtils.waitForSendingRequest();
}
```

After

```
task ("Go to the site page to visit the blog") {
	ACUtils.navigateToSitePage(
		actionType = "View Blog",
		documentTitleList = "Blogs AC Title",
		pageName = "AC Page",
		siteName = "Site Name");
}
```